gandalf.js

```diff
@@ -1,18 +1,20 @@
-const largestNumber = Math.max(...numbers);
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?